lambdas/session/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandler.java

```diff
@@ -45,7 +42,7 @@ public APIGatewayProxyResponseEvent handleRequest(
         String responseBody = "{}";
         Map<String, String> responseHeaders = Map.of("Content-Type", "application/json");
         String sessionId = input.getHeaders().get(HEADER_SESSION_ID);
-        KBVSessionItem kbvSessionItem = storageService.getSessionId(sessionId);
+        KBVSessionItem kbvSessionItem = storageService.getSessionId(sessionId).orElseThrow();
         QuestionState questionState = null;
         try {
```

Comment:
There seems to be too many try blocks here maybe just one would be ok. It have multi catch blocks if needed